app/utils/index.ts

```diff
@@ -1,21 +1,28 @@
-class Generator {
+class SequentialIntegerGenerator {
+  private START: number = 0;
+  private END: number = Infinity;
+  private STEP: number = 1;
+
   private *generate(start: number, end: number, step: number) {
     for (let i = start; i <= end; i += step) {
       yield i + step;
     }
   }
 
   private getConfig(config?: IDGeneratorConfig): Required<IDGeneratorConfig> {
-    let start: number = 0;
-    let end: number = Infinity;
-    let step: number = 1;
-
-    if (config) {
-      end = config.end || end;
-      step = config.step || step;
-      start = config.start || start;
+    if (!config) {
+      return {
+        end: this.END,
+        step: this.STEP,
+        start: this.START
```

Comment:
Wanted to ignore this but it is now looking as if we would need to make this an object, no?